### ISSUE TITLE
fix: segmentation fault on start

### ISSUE THREAD

**Issue by piegamesde** [cerebroapp/cerebro#494]
- [x] I am on the [latest](https://github.com/KELiON/cerebro/releases/latest) Cerebro.app version
- [x] I have searched the [issues](https://github.com/KELiON/cerebro/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Arch Linux
- **Cerebro.app version**: 0.3.2

## Issue

When I start cerebro, it crashes:

```
installed: X-AppImage-BuildId=847a0770-d98f-11a7-0ebd-e5b83ffea09e image: X-AppImage-BuildId=847a0770-d98f-11a7-0ebd-e5b83ffea09e
Failed to get crash dump id.
Report Id: 
[1]    3603 segmentation fault (core dumped)  ./cerebro-0.3.2-x86_64.AppImage
```
I get the same error when running the AUR packages.

**Comment by reginastracii** [comment]
similar situation, also Arch

**Comment by fxdave** [comment]
Same to me on Arch.
I also cannot install by source because nodobjc package fails to build.

**Comment by vesley** [comment]
Same to me on Arch.
When I try to start cerebro in the console as follow:

cerebro --help
cerebro -h

it starts after few retries and it works as expected, but the context menu is not readable. 
![image](https://user-images.githubusercontent.com/20923593/63223438-9a17c080-c1b5-11e9-9aae-2e85a5f5bdc6.png)

Maybe some unicode issues?

**Comment by rollingflamingo** [comment]
Same to me (Arch), mine won't event initialize config file

**Comment by fxdave** [comment]
Since there's no launcher satisfying my needs, I started my own.
Take a look:
https://github.com/vonal-project

It's still useless, there're no plugins, cannot be configured by files. But it is actively developed!

**Comment by rollingflamingo** [comment]
I managed to make it work on manjaro 18.04 by compiling from sources (version 0.3.1), I followed instruction on README.md but i before each `yarn` I run `yarn upgrade`.

(node version 8.16 AUR node-lts-carbon)

**Comment by vesley** [comment]
I switched to [ulauncher](https://ulauncher.io/). Seems to be a good launcher too.

**Comment by williamcanin** [comment]
**Same problem here on Arch Linux. In the AUR version, it asks for nodejs-lts-carbon and I use a newer nodejs.**

**Comment by cyberquarks** [comment]
```
$ yay -S cerebro
:: There are 2 providers available for cerebro:
:: Repository AUR
    1) cerebro 2) cerebro-git 

Enter a number (default=1): 1
 -> Could not find all required packages:
        nodejs-lts-carbon (Wanted by: cerebro)
```

**Comment by apertureless** [comment]
I have the same issue on kali linux

**Comment by Jay-flow** [comment]
I have the same issue on endeavour os

**Comment by dubisdev** [comment]
I'm closing this issue as outdated. We recently released `v0.8.0` which has a lot of dependency updates and bug fixes. Try it and, if the problem persists, we can open a new issue.
